### PR TITLE
New Neko Console Commands  Cheat Sheet

### PR DESCRIPTION
**Pull request by fzzr-**
IA: https://duck.co/ia/view/neko_cli_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@fzzr- Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by fzzr-** [comment]
Hey, Travis!? It's not mine fail, please!

File: share/goodie/cheat_sheets/json/neko/neko-cli.json [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
+  "id": "neko_cli_cheat_sheet",
+  "name": "Neko VM",
+  "version": "2.0.0",
+  "description": "Neko has a compiler and a virtual machine. The Virtual Machine is both very lightweight and well optimized, so it can run very quickly. The VM can be easily embedded into any application and your libraries can be accessed using the C foreign function interface.",
```
Comment (fzzr-) [inline share/goodie/cheat_sheets/json/neko/neko-cli.json]: Have to shorten too?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Neko cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/neko_vm_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Neko cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/neko_vm_mobile.png)

**Comment by fzzr-** [comment]
This need rename too? Because IA Page ID `neko_vm` not equal `neko_cli`.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Neko cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/neko_vm_mobile.png)

File: share/goodie/cheat_sheets/json/neko/neko-cli.json (hunk 1 above)
Comment (fzzr-) [inline share/goodie/cheat_sheets/json/neko/neko-cli.json]: * Shortened description;
* removed `version`

Finally need rename...

File: share/goodie/cheat_sheets/json/neko/neko-cli.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+  "metadata": {
+    "sourceName": "Neko VM Documentation",
+    "sourceUrl": "http://nekovm.org/doc/tools"
+  },
+
+  "aliases": [
+    "neko",
+    "nekoc",
+    "nekoc",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/neko/neko-cli.json]: @fzzr-  above two aliases are same

Comment (fzzr-) [inline share/goodie/cheat_sheets/json/neko/neko-cli.json]: fixed, sorry :(

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Neko cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/neko_vm_mobile.png)

**Comment by moollaza** [comment]
@fzzr- Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@fzzr- this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**